src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -97,6 +99,7 @@ private Test find(String uuid) throws LightblueException {
         DataFindRequest findRequest = new DataFindRequest(ENTITY);
         findRequest.select(Projection.includeFieldRecursively("*"));
         findRequest.where(Query.withValue("_id", Query.eq, uuid));
+        findRequest.execution(Execution.MongoController.withReadPreference(ReadPreference.primary));
```

Comment:
@paterczm - is there a way we could shortcut this call? Maybe just Execution.withReadPreference(...);